temporal-testing/src/test/java/io/temporal/testing/TestWorkflowEnvironmentSleepTest.java

```diff
@@ -79,22 +85,48 @@ public void setUp() {
     worker = testEnv.newWorker(WORKFLOW_TASK_QUEUE);
     client = testEnv.getWorkflowClient();
     worker.registerWorkflowImplementationTypes(ExampleWorkflowImpl.class);
-    workflow =
-        client.newWorkflowStub(
-            ExampleWorkflow.class,
-            WorkflowOptions.newBuilder().setTaskQueue(WORKFLOW_TASK_QUEUE).build());
     testEnv.start();
   }
 
+  @After
```

Comment:
You can refactor this to use SDKTestWorkflowRule, then setting up clients, workers @Before & @After by hand are not needed.